lib/cusparse/linalg.jl

```diff
@@ -48,6 +48,66 @@ function sum_dim2(A::CuSparseMatrixCSR{T}) where {T}
     return colsum
 end
 
+function compute_threads(max_threads, wanted_threads, shuffle, dev)
+    if wanted_threads > max_threads
+        shuffle ? prevwarp(dev, max_threads) : max_threads
+    else
+        wanted_threads
+    end
+end
```

Comment:
That's a very generic name for a function. Either inline it in the calling function, or give it a more specific name.